---
title: "Final Report: Effects Of Kibble And Raw Meat Diets With Or Without Additives On Fecal Microbial Communities Of Healthy Adult Beagles"
author: "Emre Ovet"
date: "December 5, 2018"
output:
    github_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design

In the research study of Beloshapka et al., 6 spayed female, healthy adult beagle dogs with ages varying from 5 to 6 were used. Six raw diets, based on animal meats and organs, eggs, fruits, and vegetables, were formulated by Beloshapka et al to contain approximately 25–30% crude protein and 45–50% fat [@beloshapka2013fecal]. The formulated diets were beef control, beef with 1.4% inulin dry matter basis, beef with 1.4% yeast cell wall, chicken control, chicken with 1.4% inulin and chicken with 1.4% yeast cell wall. Dogs were fed twice daily, first at 8:00 AM and then at 17:00 PM with either formulated raw diets or kibble. The diets were truly raw and were not treated for pathogens and they were kept frozen and thawed under refrigeration in storage containers 2 days prior to feeding. Subsamples of all six thawed diets were submitted to the University of Illinois Veterinary Medicine Diagnostic Laboratory for general cultures, all of which came back negative or below detectible limits [@beloshapka2013fecal].

## Sample origin and sequencing

Following a 14-day adaptation period, one fresh fecal sample was collected by Beloshapka et al. within 15 min of defecation on either day 15 or day 16 of each period. The fresh feces were weighed and aliquoted into sterile cryogenic vials and frozen at −80 °C until DNA extraction [@beloshapka2013fecal].

Genomic DNA was extracted by Beloshapka et al. using the repeated bead beater method with QIAamp DNA Stool Mini DNA extraction Kit. Once the DNA was eluted, all samples were purified using the QIAquick PCR Purification Kit. Extracted DNA was quantified using NanoDrop ND-1000 spectrophotometer. Genomic DNA quality was assessed using agarose gel electrophoresis [@beloshapka2013fecal]. Genomic DNA was diluted to 20 ng μL−1 in preparation for 454-pyrosequencing. Bacterial tag-encoded FLX-Titanium amplicon pyrosequencing based upon the V4-V6 region of the 16S rRNA gene was performed. The 16S rRNA universal Eubacterial primers 530F and 1100R were used to amplify the 600-bp region. In preparation for FLX-Titanium sequencing, DNA fragment size and concentration were accurately measured using DNA chips under a Bio-Rad Experion Automated Electrophoresis Station and a TBS-380 Fluorometer. A four-region 454 sequencing run was performed on a GS PicoTiterPlate using the Genome Sequencer FLX System. 

Beloshapka et al. used quantitative polymerase chain reaction to measure the abundance of *Bifidobacterium* and *Lactobacillus*. Fecal samples were extracted using ZR Fecal DNA Extraction Kit along with 
SYBR-based reaction mixtures. PCR conditions were 95°C for 2 minutes and 40 cycles at 95°C for 5 seconds and 10 seconds at 60°C for *Bifidobacterium* and were 58°C for *Lactobacillus*. A melt curve analysis was performed for 1 minute at 95 °C, 1 minute at 55 °C, and 80 cycles of 0.5°C increments each 10 seconds. The PCR conditions were 95 °C for 20 seconds, 40 cycles at 95 °C for 5 seconds, and 10 seconds at the optimized annealing temperature.

## Computational

I downloaded fastq files from NCBI with accession code PRJNA186650. Then, I installed general-use packages from GitHub and bioconductor. I set the base path for my input data files from fecal microbial community study, sorted the path to ensure samples are in order, extracted sample names and specified the full path for each of the filenames forward reads. Next, I used DADA2, a software package that corrects and models Illumina-sequenced amplicon errors to plot the quality profiles of each sample [@callahan2016]. After checking those quality profiles,I created a diretory called "filtered_reads_path", placed my filtered files inside the filtered/ subdirectory and trimmed the lower quality sequences using DADA2 [@callahan2016]. Then, I generated a table of read counts before and after trimming. This was followed by building error models from each of the samples and visualizing errors with plots to make sure error models matched my data. After checking the plots, I removed all the replicated sequences to increase DADA2's accuracy [@callahan2016]. Next, I named the dereplicated class objects by the sample names. I ran DADA to apply the core sample inference algorithm to the dereplicated data [@callahan2016]. After DADA2 ran succesfully, I created an histogram of sequence lenghts to check the distribution of trimmed and denoised sequences. Then, I removed the chimeras from my cleaned reads to obtain just non-chimeric reads. After that, I builded a table of pipeline read counts to see how many sequences remain at each step of the pipeline. Next, I assigned taxonomy to each sequence variant. After assigning taxonomy, I extracted sequences to Fasta and builded a phylogeny using DADA2 [@callahan2016]. Finally, I read in the phylogeny and construct a phyloseq object by using the phyloseq package, a tool to import, store, analyze, and graphically display complex phylogenetic sequencing data [@mcmurdie2013]. After completing all these steps, I melted and saved the phyloseq. For generating plots, I used ggplot. 


# Results

## Subsections are ok in the results section too

```{r load-packages-and-data}
# load packages as needed
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("citr")
library("dada2")
library("seqinr")
library("mctoolsr")
library("phyloseq")

load("output/phyloseq_obj.Rdata")
load("output/melted_phyloseq.Rdata")
# load data produced from analysis scripts using
# something like load("output/processed_data.Rdata")
```

```{r}
melted_phyloseq %>%
  ggplot(aes(x = Family, y = Abundance, fill = food)) +
  geom_col(position = position_dodge()) +
  theme(axis.text = element_text(angle = 90, hjust = 1)) +
ggtitle("Abundance of Bacterial Family by Diet")
```

```{r}
melted_phyloseq %>%
  filter(food %in% c("chicken")) %>%
ggplot(aes(x = Family, y = Abundance, fill = additive)) +
  geom_col(position = position_dodge()) +
  theme(axis.text = element_text(angle = 90, hjust = 1)) +
ggtitle("Abundance of Bacterial Family in
       Chicken Diet With or Without Additives")
```

```{r}
melted_phyloseq %>%
  filter(food %in% c("beef")) %>%
ggplot(aes(x = Family, y = Abundance, fill = additive)) +
  geom_col(position = position_dodge()) +
  theme(axis.text = element_text(angle = 90, hjust = 1)) +
ggtitle("Abundance of Bacterial Family in Beef Diet With or Without Additives")
```

```{r}
melted_phyloseq %>%
  ggplot(aes(x = Genus, y = Abundance, fill = food)) +
  geom_col(position = position_dodge()) +
  theme(axis.text = element_text(angle = 90, hjust = 1)) +
ggtitle("Abundance of Bacterial Genus by Diet")
```

```{r}
melted_phyloseq %>%
  filter(food %in% c("chicken")) %>%
ggplot(aes(x = Genus, y = Abundance, fill = additive)) +
  geom_col(position = position_dodge()) +
  theme(axis.text = element_text(angle = 90, hjust = 1)) +
ggtitle("Abundance of Bacterial Genus in
       Chicken Diet With or Without Additives")
```

```{r}
melted_phyloseq %>%
  filter(food %in% c("beef")) %>%
ggplot(aes(x = Genus, y = Abundance, fill = additive)) +
  geom_col(position = position_dodge()) +
  theme(axis.text = element_text(angle = 90, hjust = 1)) +
ggtitle("Abundance of Bacterial Genus in Beef Diet With or Without Additives")
```


# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
